### ISSUE TITLE
approximate size of training set

### ISSUE THREAD

**Issue by naturegirl** [zxzhijia/GoPiGo-Driven-by-Tensorflow#1]
Hey zxzhijia,
thanks for providing this really interesting example! I'm trying out something very similar where I'm training a GoPiGo car to drive along a route with a tensorflow model. (Without using the inception model though)

One qusetion. For your data collection, how many images have you collected approximately for your training dataset?

Thank you!

**Comment by zxzhijia** [comment]
About 400 images for each label. In your case, I think that depends on the size of your network.

**Comment by naturegirl** [comment]
Thank you!

**Comment by zxzhijia** [comment]
No problem.
